### DISCUSSION TITLE
How can I track the meal plan before today?

### DISCUSSION THREAD

**Discussion by geniusshiun** [mealie-recipes/mealie#2415] [Q&A] [answered]
I'd like to use mealie to track my meal history. Can I do it?

**Comment by michael-genson** [comment, marked as answer]
Mealie has Meal Plans which let you plan out meals with recipes. Those are stored indefinitely, so it creates a nice history log.
In addition, there's a "Last Made" feature which lets you post every time you make a recipe, which you can then [access on a timeline](https://github.com/hay-kot/mealie/pull/2265)

Note that if you use meal plans, the timeline is automatically updated on the day of the meal, so you don't have to do both!
